citations/stat.js

```diff
@@ -29,13 +29,12 @@ module.exports = {
   ],
 
   links: function(cite) {
-    // GPO only has volumes starting with 65.
-    if (cite.volume < 65) return { };
+    var links = { };
 
-    var usgpo_url = "http://api.fdsys.gov/link?collection=statute&volume=" + cite.volume + "&page=" + cite.page;
-
-    return {
-      usgpo: {
+    // GPO
+    if (cite.volume >= 65 && cite.volume <= 125) {
```

Comment:
Presumably eventually they'll publish more volumes. Their latest volume (125) is from laws enacted in 2011, so they seem to take their time in publishing this.